### PR TITLE
Admin document manager - test case part 1

### PR DESCRIPTION
**Pull request by RuchetaG**

### CODE REVIEW

File: hw-e2e/admin-document-manager.spec.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  await page.goto(`${baseURL}`);
+  const sideMenu = new SideMenu(page);
+  await sideMenu.openAdministrativeDocumentManager();
+  const administrativeDocumentManager = new AdministrativeDocumentManager(page);
+  await administrativeDocumentManager.uploadFileFromGivenPath("./README.md");
+  await administrativeDocumentManager.openUnsupportedFile();
+  await administrativeDocumentManager.validateNoPreviewText();
+});
+
+test("Validate folder label for the category drop down", async ({
```
Comment (pavankumar017) [inline hw-e2e/admin-document-manager.spec.ts]: Didnt Get this scenario ?

File: hw-e2e/admin-document-manager.spec.ts [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  const administrativeDocumentManager = new AdministrativeDocumentManager(page);
+  await administrativeDocumentManager.validateSearchPlaceholder();
+});
+
+test("Validate search in folder category", async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}`);
+  const sideMenu = new SideMenu(page);
+  await sideMenu.openAdministrativeDocumentManager();
+  const administrativeDocumentManager = new AdministrativeDocumentManager(page);
+  await administrativeDocumentManager.validateSearchInFolderCategory();
```
Comment (pavankumar017) [inline hw-e2e/admin-document-manager.spec.ts:97]: Shouldn't we have seperate step to enter a keyword in search field and then Validate search - This applies Eveywhere

File: hw-e2e/admin-document-manager.spec.ts (hunk 1 above)
Comment (RuchetaG) [reply hw-e2e/admin-document-manager.spec.ts]: Updated

File: locators/administrative-document-manager.ts [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+    this.heading = this.page.getByRole("heading", { name: "Document Manager" });
+    this.noPreviewText = this.page.getByText("No preview available");
+    this.uploadCategory = this.page.locator('[data-testid="folder"]');
+    this.categoryOptions = ["Inbox", "Audit", "Education", "Administrative"];
+    this.uploadModalUploadButton = this.page
+      .getByTestId("upload-modal")
+      .getByRole("button", { name: "Upload" });
+    this.uploadInput = this.page.getByTestId("upload-patient-documents");
+    this.search = this.page.getByPlaceholder("Search");
+    this.unsupportedFileSearch = this.page.getByText("README.md").first();
```
Comment (pavankumar017) [inline locators/administrative-document-manager.ts:48]: this first() method - does it selects the first element found?

Comment (RuchetaG) [reply locators/administrative-document-manager.ts:48]: yes

File: hw-e2e/admin-document-manager.spec.ts (hunk 2 above)
Comment (RuchetaG) [reply hw-e2e/admin-document-manager.spec.ts:97]: updated

**Review by pavankumar017 — APPROVED** [review]

**Review by pankaj51 — APPROVED** [review]

**Review by MohammedMoosaSait — APPROVED** [review]

**Pull request merged**